src/api/app/models/workflow/step.rb

```diff
@@ -15,7 +15,9 @@ def call(_options)
   end
 
   def target_project_name
-    "home:#{@token.user.login}:#{source_project_name}:PR-#{scm_webhook.payload[:pr_number]}"
+    return step_instructions[:target_project] if scm_webhook.push_event?
+
+    "#{step_instructions[:target_project]}:#{source_project_name}:PR-#{scm_webhook.payload[:pr_number]}"
```

Comment:
While we are at it we wanted to get off `source_project_name` here right? So more like

```ruby
"#{step_instructions[:target_project]}:#{scm_webhook.payload[:target_repository_full_name].split('/').join(':')}:PR-#{scm_webhook.payload[:pr_number]}"
```

Reply:
Alright, I overlooked that the naming scheme for pull request also changes, going to adapt it. Looking at the wiki now the naming looks a bit different `OBS:Server:Unstable:CI:PR-$MY_PR_NUMBER/obs-server`, but I like the way you described. So I would go for it

Reply:
I changed the wiki in the meanwhile ;-)

Reply:
@hennevogel Alright, so changed the code and test to use the new naming schema for PR's